perfscale_regression_ci/scripts/network/network_policy_scalability.sh

```diff
@@ -17,66 +17,70 @@ source ../common.sh
+export WORKLOAD_TEMPLATE=$workload_template
+run_workload
+
 for ((i = 0; i < 2; i++));
-do
-    export WORKLOAD_TEMPLATE=${workload_template_list[$i]}
-    export NAME=${kube_burner_job_list[$i]}
-    export NAMESPACE=${kube_burner_job_list[$i]}
-    deployment_name=${kube_burner_job_list[$i]}
+do 
     scale_up=0
-
-    echo "Starting scale-up test for ${kube_burner_job_list[$i]}..."
-    run_workload
     oc scale --replicas $pod_scale_max deployment $deployment_name -n ${NAMESPACE}; start_time=`date +%s`
     echo "start time: $start_time"
     scale_up=$(oc get deployment $deployment_name --no-headers -n ${NAMESPACE} | awk -F ' {1,}' '{print $4}')
     check_deployment_pod_scale ${NAMESPACE} $deployment_name $scale_up $pod_scale_max
     end_time=`date +%s`
     echo "end time: $end_time"
     final_time=$((end_time - start_time))
-    echo "execution time for test ${kube_burner_job_list[$i]}: $final_time s."
+    echo "execution time for test ${kube_burner_job_list}: $final_time s."
     final_time_list+=($final_time)
-	sleep 1
-    echo "=========================================================================="
+	oc delete networkpolicy --all -n ${NAMESPACE}
+	oc get networkpolicy -n ${NAMESPACE}
+	if [[ $i -eq 0 ]]; then
+		# want to scale down first deployment to be able to 
+		oc scale --replicas 500 deployment $deployment_name -n ${NAMESPACE}
+		check_deployment_pod_scale ${NAMESPACE} $deployment_name $pod_scale_max 500
+		sleep 10
+		echo "=========================================================================="
+	fi
 done
 
-sleep 5
-
 pod_name=$(oc get po -n ${NAMESPACE} --no-headers | head -n 1 | awk '{print $1}')
 pod_ip=$(oc get po $pod_name -n  ${NAMESPACE} --output jsonpath='{.status.podIP}')
 apiserver_pod=$(oc get po -n openshift-oauth-apiserver --no-headers | head -n 1 | awk '{print $1}')
 
 echo "Starting test to deny network traffic in the namespace ${NAMESPACE}..."
-oc apply -f $deny_network_policy -n ${NAMESPACE}
-deny_time_start=`date +%s`
+oc create -f $deny_network_policy -n ${NAMESPACE} && deny_time_start=`date +%s`
 check_http_code $deny_traffic_code $pod_ip $apiserver_pod
 deny_time_end=`date +%s`
 deny_final_time=$((deny_time_end - deny_time_start))
 echo "Time taken for network policy to become inactive: $deny_final_time s"
 
-sleep 1
-
 echo "Starting test to allow network traffic in the namespace ${NAMESPACE}..."
-oc delete networkpolicy deny-by-default -n ${NAMESPACE}
-return_traffic_start=`date +%s`
+oc delete networkpolicy deny-by-default -n ${NAMESPACE} && return_traffic_start=`date +%s`
 check_http_code $return_traffic_code $pod_ip $apiserver_pod
 return_traffic_end=`date +%s`
 return_traffic_final_time=$((return_traffic_end - return_traffic_start))
 echo "Time taken for network policy to become active: $return_traffic_final_time s"
 
-echo "=========================================================================="
-
 
 final_time_np=${final_time_list[0]}
 final_time_no_np=${final_time_list[1]}
```

Comment:
@paigerube14 Why do you want to remove the "no-np-issue-test"? If it is removed, I think this line should be removed too.